src/actions/user.actions.js

```diff
@@ -1,62 +1,39 @@
 import { userConstants } from '../constants/user.constants';
 import { userService } from '../services/user.services';
 import { history } from '../helpers/history';
-import { alertActions } from '../actions/alert.actions';
 import { handleLoginError, handleSignupError } from '../helpers/error.handler';
 import { homePageLink, loginPageLink } from '../constants/link.constants';
 
-export const request = (user) => {
-  return { type: userConstants.USER_REQUEST, user };
+export const request = () => {
```

Comment:
I suggest using the same names for actions functions, in this way you can search an action more easily
```suggestion
export const userRequest = () => {
```

Reply:
Also in JS you can return implicitly (functional paradigm fan here)